engine/Shopware/Controllers/Backend/Analytics.php

```diff
@@ -276,11 +276,17 @@ public function getRatingAction()
             }
         }
 
+        // Sets the correct limit
+        $limit = 25;
```

Comment:
```suggestion
        $limit = (int) $this->Request()->getParam('limit', 25);
```

You should use the request parameter as initial value. Otherwise the user cannot select a larger page size.

Reply:
$this->Request()->query->getInt('limit', 25);